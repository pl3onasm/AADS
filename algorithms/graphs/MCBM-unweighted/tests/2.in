1 1

a f 1
b f 1
b h 1
c g 1
c h 1
c i 1
d h 1
e h 1