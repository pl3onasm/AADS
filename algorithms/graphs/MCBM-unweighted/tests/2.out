
--------------------
 Network
 Vertices: 9
 Edges: 8
--------------------
  a[1]: f(1)
  b[2]: h(1), f(1)
  c[3]: i(1), h(1), g(1)
  d[1]: h(1)
  e[1]: h(1)
  f[0]
  g[0]
  h[0]
  i[0]
--------------------

---------------
 Matching
---------------
  c -- i
  b -- f
  e -- h
---------------